lib/node_modules/@stdlib/array/complex128/lib/main.js

```diff
@@ -552,6 +565,75 @@ setReadOnly( Complex128Array, 'of', function of() {
 	return new this( args );
 });
 
+/**
+* Returns an array element with support for both nonnegative and negative integer indices.
+*
+* @name at
+* @memberof Complex128Array.prototype
+* @type {Function}
+* @param {integer} idx - element index
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} must provide an integer
+* @returns {(Complex128|void)} array element
+*
+* @example
+* var arr = new Complex128Array( 10 );
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
```

Comment:
```suggestion
* var real = require( '@stdlib/complex/real' );
* var imag = require( '@stdlib/complex/imag' );
*
* var arr = new Complex128Array( 10 );
```

Invalid import order and need to update the import statements and function calls below.